### DISCUSSION TITLE
LAN connection

### DISCUSSION THREAD

**Discussion by ColonelBlimp** [winder/Universal-G-Code-Sender#1745] [Help] [answered]
Hi, can UGS connect (from PC to controller) via LAN?

Many thanks.

**Comment by breiler** [comment, marked as answer]
It depends on the controller, but yes it can connect to a LAN based controller.  Though it is a bit cumbersome to make the settings, there are some instructions here: https://github.com/winder/Universal-G-Code-Sender/wiki/Connecting-the-Controller#grbl--network

**Reply by ColonelBlimp** [reply]
Many thanks for the reply.  Appreciated :-)
